examples/player/simid_player.js

```diff
@@ -144,8 +157,12 @@ class SimidPlayer {
     // The target of the player to send messages to is the newly
     // created iframe.
     playerDiv.appendChild(simidIframe);
-    // Set up css to overlay the SIMID iframe over the video creative.
-    simidIframe.classList.add('simid_creative');
+    // Set up css to overlay the SIMID iframe over the entire video creative
+    // only if linear.
+    if (this.isLinearAd_){
+      simidIframe.classList.add('simid_creative');
```

Comment:
Is there any css needed for a non linear creative?

Reply:
No, it will instead be positioned by the given dimension inputs